API/SearchService/__init__.py

```diff
@@ -8,15 +8,29 @@
 from .src.exception import ParameterException
 from .src.search_service import get_search_results
 
-from .src.util import MatchRule, SearchScope, append_results, build_or_search_statement, build_search_statement, get_param_int, get_param_match_rule, get_param_search_scope, get_param_str
+from .src.util import MatchRule, SearchScope, append_results, build_or_search_statement, build_search_statement, get_param_int, get_param_match_rule, get_param_search_scope, get_param_str, get_param_list
+from ..ChatgptService import ask_chatgpt_service
 
+# nl2keyword process switch
+nl2keyword = os.environ["ENABLE_NL2KEYWORD"]
 
 def main(req: func.HttpRequest,
          context: func.Context) -> func.HttpResponse:
     logging.info('Python HTTP trigger function processed a request.')
 
     try:
         keyword = get_param_str(req, "keyword", required=True)
+        if nl2keyword.lower() == "true":
+            question = user_msg = keyword
+            history_msg = get_param_list(req, "history_msg")
+            openai_prompt = json.loads(os.environ.get('OPENAI_NL2KEYWORD_MSG'))
```

Comment:
Suggest naming this variable more accurately to increase readability
```suggestion
            extract_keywords_prompt = json.loads(os.environ.get('OPENAI_NL2KEYWORD_MSG'))
```

Reply:
Same as above https://github.com/hackathon-cli-recommendation/cli-recommendation/pull/34/files#r1295587006